.github/workflows/elixir.yml

```diff
@@ -47,5 +47,11 @@ jobs:
       run: mix deps.get
     - name: Run tests
       run: mix test
+      env:
+        CLIENT_ID: :sfractal2020
```

Comment:
is this a default or a hardcoded value? 
Eitherway, can we use :sbompoc for this repo (ie don't use same for each repo)?

Reply:
This is for use in test environment only on github